src/workers/parser/getSymbols.js

```diff
@@ -118,7 +118,7 @@ function getSpecifiers(specifiers) {
 
   return specifiers.map(specifier => specifier.local && specifier.local.name);
 }
-
+/* eslint-disable complexity */
```

Comment:
Adding this because `extractSymbol` has the potential of being more complex the complexity limit for 23.